### DISCUSSION TITLE
Kernelspec jupyter notebook not working

### DISCUSSION THREAD

**Discussion by Rizwankaka** [AammarTufail/python_ka_chilla2023#62] [Q&A]
![IMG_20230114_005224_497](https://user-images.githubusercontent.com/122643396/212410074-9c1c5b73-c292-43f8-b70b-25cdacfeee30.jpg)

**Comment by Rizwankaka** [comment]
Can anyone solve this issue, i have been struggling with this error..using win 8.1 and python 3.8. I installed and reinstalled python and vscode twice still its occuring

**Reply by usamabajwa86** [reply]
using Python 3.8.3. delete  folder the C:\Users[USERNAME]\AppData\Roaming\Python directory and then reinstall all python extensions in VS.

**Reply by Rizwankaka** [reply]
Thanks bro I uninstalled python 3.8 and did installed 3.6 and updated all via pip and problem resolved... thanks Osama for your valuable comment and support. I really appreciate it.
Big thanks

**Reply by usamabajwa86** [reply]
kindly push answered botton
